pkg/provider/aws/plugin/loadbalancer/logger.go

```diff
@@ -0,0 +1,21 @@
+package aws
```

Comment:
Do I still need this file?

Reply:
I'd leave it for now.   This looks like a AWS api thing.

Reply:
ok